providers/github.go

```diff
@@ -198,6 +206,43 @@ func (p *GitHubProvider) hasOrgAndTeam(accessToken string) (bool, error) {
 	return false, nil
 }
 
+// ValidateUser validates that the user exists as a collaborator on the specified repo
+func (p *GitHubProvider) ValidateUser(user string) bool {
+	//https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
```

Comment:
Nit: I would typically expect a space here
```suggestion
	// https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
```